backend/app/controllers/uoas.js

```diff
@@ -68,17 +68,26 @@ module.exports = {
     },
 
     insert: function (req, res, next) {
+        //TODO: Fix test to pass for this
```

Comment:
This will be fixed in my next ticket. @jmfadeley We should probably start making sure all tests pass before we merge into develop. Or at the very least, the tests that have been directly affected by what we are working on. 

It seems like after this function was worked on, the `task.integration` and `discussion.integration` tests started failing as a result of a change from here. I will look into it more in my next ticket. You can ignore the `console.log()` statements for now.